cloud/deployment/inspect/inspect.go

```diff
@@ -29,6 +29,7 @@ type deploymentMetadata struct {
 	UpdatedAt           *time.Time           `mapstructure:"updated_at" yaml:"updated_at" json:"updated_at"`
 	DeploymentURL       *string              `mapstructure:"deployment_url" yaml:"deployment_url" json:"deployment_url"`
 	WebserverURL        *string              `mapstructure:"webserver_url" yaml:"webserver_url" json:"webserver_url"`
+	APIURL              *string              `mapstructure:"api_url" yaml:"api_url" json:"api_url"`
```

Comment:
Should we call this AirflowApiUrl to not confuse with astronomer apis

Reply:
Can do -- everything else in this context is talking about the deployment specifically so it felt a bit redundant to me.

I'll defer to other's view on this.

Reply:
I do think some customers confuse the Airflow API and our API. May be helpful to specify that this is the API for Airflow